soa/types/date.h

```diff
@@ -13,6 +13,7 @@
 #include "boost/date_time/posix_time/posix_time.hpp"
 #include "jml/utils/parse_context.h"
 #include "jml/db/persistent_fwd.h"
+#include <time.h>
```

Comment:
I guess you wanted to include "time.h" and not <time.h> ? (the system header)

Why do you need this header in date.h ?